www/src/pages/components/image/react/index.mdx

```diff
@@ -92,6 +93,44 @@ This example uses the `height` prop to "crop" the image. By default, the cropped
 />
 ```
 
+## Forcing early render
+
+In some cases, like server-side rendering, you want the HTML to be complete without the need for client-side javascript. The `forceEarlyRender` prop overrides the default `Image` behavior that would both calculate the `sizes` attribute and lazy load the image.
```

Comment:
```suggestion
The `Image` component behavior uses JavaScript to lazy load the image and and calculate the `sizes` attribute. While this is beneficial in most cases, it also means that the image will not start downloading until the client-side JavaScript is parsed and the image is scrolled into view. The `forceEarlyRender` prop allows developers to turn off lazy loading and the `sizes` calculation so that the image can load as soon as possible. This is useful for "above-the-fold" images with a predictable width in server-side-rendered environments.
```

Clarifying that it's not something you _need_ in server-side rendering. Only if it's "above-the-fold" and the width is predictable.